go/vt/proto/query/query.pb.go

```diff
@@ -1283,6 +1283,7 @@ type ExecuteOptions struct {
 	// TransactionAccessMode specifies the access modes to be used while starting the transaction i.e. READ WRITE/READ ONLY/WITH CONSISTENT SNAPSHOT
 	// If not specified, the transaction will be started with the default access mode on the connection.
 	TransactionAccessMode []ExecuteOptions_TransactionAccessMode `protobuf:"varint,14,rep,packed,name=transaction_access_mode,json=transactionAccessMode,proto3,enum=query.ExecuteOptions_TransactionAccessMode" json:"transaction_access_mode,omitempty"`
+	SessionUUID           string                                 `protobuf:"bytes,15,opt,name=SessionUUID,proto3" json:"SessionUUID,omitempty"`
```

Comment:
I'm not too happy about adding the `SessionUUID` here, but I couldn't find a better way to pass it through to the tablet selection logic. If anyone has a better idea how that could be solved, I'm all ears. 🙇‍♂️

Reply:
Another way is to pass SessionUUID explicity to all the methods.